packages/xo-web/src/xo-app/xoa/update/index.js

```diff
@@ -186,6 +229,16 @@ const Updates = decorate([
           .sort()
           .map(name => `- ${name}: ${installedPackages[name]}`)
           .join('\n'),
+      xoaReleaseChannelsOptions: ({ xoaReleaseChannels }) => [
+        ...map(xoaReleaseChannels, elt => ({
+          label: elt.description,
```

Comment:
The description is a string describing the channel, it may possibly be long.

What the user needs to see is the channel id (eg `stable`).

Reply:
This is spec a problem, I was told that the `id` will be a generated UUID